app/templates/model-control-menu.html

```diff
@@ -41,6 +41,52 @@
         </a>
       </li>
 
+      <li role="separator" class="divider"></li>
+
+      <li :class="{disabled: numSelectedModels === 0  || !someSelectedModelsAreFinished}">
```

Comment:
looks a bit repetitive. Can we make all the download options configurable somehow?